scripts/service-accounts/generateServiceAccounts.go

```diff
@@ -38,42 +45,130 @@ var (
 		"dp-api-router",
 		"dp-download-service",
 		"dp-frontend-geography-controller",
-		"dp-identity-api"}
+		"dp-identity-api",
+	}
 )
 
+type ServiceAccount struct {
+	ID   string `json:"-"`
+	Name string `json:"id,omitempty"`
+}
+
+func init() {
+	s1 := rand.NewSource(time.Now().UnixNano())
+	r = rand.New(s1)
+}
+
 func main() {
+	serviceDir := flag.String("dir", "service", "the service account directory path")
```

Comment:
The instructions say it should be `/service` but the default here doesn't have the `/` - intentional difference?